alarms.tf

```diff
@@ -134,11 +134,41 @@ resource "aws_cloudwatch_metric_alarm" "swap_usage_too_high" {
   period              = var.period
   statistic           = "Average"
   threshold           = local.thresholds["SwapUsageThreshold"]
-  alarm_description   = "Average database swap usage over last ${var.period/60} minutes too high, performance may suffer"
+  alarm_description   = "Average database swap usage over last ${var.period / 60} minutes too high, performance may suffer"
   alarm_actions       = [var.aws_sns_topic_arn]
   ok_actions          = [var.aws_sns_topic_arn]
 
   dimensions = {
     DBInstanceIdentifier = var.db_instance_ids[count.index]
   }
 }
+
+resource "aws_cloudwatch_metric_alarm" "rapid-free-space-decrease" {
+  count                     = length(var.db_master_ids)
+  alarm_name                = "${var.name_prefix}${var.db_master_ids[count.index]}-rapid-free-space-decrease"
+  comparison_operator       = "LessThanOrEqualToThreshold"
+  evaluation_periods        = "1"
+  threshold                 = "-3000000000"
```

Comment:
The threshold should be a function of the database size. Small databases should have a lower threshold, large databases a higher threshold.